rom/dev/src/rom_env.rs

```diff
@@ -72,6 +74,9 @@ pub struct RomEnv {
 
     /// FHT Data Store
     pub fht_data_store: FhtDataStore,
+
+    /// Cryptographically Secure Random Number Generator
+    pub csrng: Csrng,
```

Comment:
This won't work with the default verilator; you need to use the Trng type here.

Reply:
Then it's just a call to trng.generate() to get a 384-bit value, and it will come from either the etrng or the itrng depending on how the hardware was built.

Reply:
Updated